R/df_to_list.R

```diff
@@ -0,0 +1,8 @@
+df_to_list <- function(df) {
```

Comment:
Is this file generated?

Reply:
@Marc-Andre-Rivet This function was also user-authored (by me), to emulate the syntax on the Python side using `df_to_dict`. R users reported that it made using `dashDataTable` easier for them, so it's persisted. It turns out that it's useful for `dashBio` as well.

Reply:
I think this one is OK to leave in `master`, along with the rest of the R package items (vignettes, Rd files, etc).